ROADMAP.md

```diff
@@ -0,0 +1,118 @@
+
+
+# go-libp2p roadmap Q4’22/Q1’23
+
+```
+Date: 2022-10-07
+Status: In Progress
+Notes: This document is still in review will may be heavily modified based on stakeholder feedback. Please add any feedback or questions in:
+https://github.com/libp2p/go-libp2p/issues/1806
+```
+
+## 🛣️ Milestones
+### 2022
+
+#### Early Q4 (October)
+-   A.1⚡Handshakes at the Speed of Light - **[Early Muxer Negotiation](https://github.com/libp2p/specs/issues/426)**
+- E.1 🧪 Future-proof testing - **RTT test for Early Muxer Negotiation**
+
+### Mid Q4 (November)
+-   F.1 ⏱ Future-proof Benchmarking - **[Using nix-builders](https://github.com/testground/testground/pull/1425)**
+
+#### End of Q4 (December)
+-   B.1 🧠 Smart Dialing - **Happy Eyeballs**
+-   C 📊 Comprehensive Metrics - **Swarm metrics**
+-   D.1 📺 Universal Browser Connectivity - **[WebRTC Browser -> Server](https://github.com/libp2p/go-libp2p/pull/1655)**
```

Comment:
I think the ordering should be:

-   D.1 📺 Universal Browser Connectivity - **[WebRTC Browser -> Server](https://github.com/libp2p/go-libp2p/pull/1655)**
-   B.1 🧠 Smart Dialing - **Happy Eyeballs**
-   C 📊 Comprehensive Metrics - **Swarm metrics**

Reason being:
1. We're not going to do anything to slow down the browser connectivity story.  This closes up the story as well on "all modern browsers can connect with a go-libp2p node". 
2. Smart dialing - assuming we actually hitting a breaking point for what transport to use

Reply:
Per comment other places I believe, I would bump smart dialing and comprehensive metrics until we're more solid on future proofing interoperability and performance.